docs/superpowers/specs/2026-08-27-issue-26-plugin-architecture-design.md

```
@@ -0,0 +1,778 @@
+> Diseño aprobado derivado de la Issue #26, `CONTEXT.md`, los ADR 0001, 0002,
+> 0003, 0004, 0006, 0007, 0008, 0009, 0010, 0011, 0013, 0014 y 0015, y el
+> trabajo previo de las Issues #11 y #17.
+
+## Introduction
+
+Esta especificación define una arquitectura para añadir nuevos `Step types`,
+registrar y seleccionar runtimes y dejar una frontera evolutiva para futuros
+`Source providers` y extensiones de terceros.
+
+El diseño no implementa la arquitectura ni construye el prototipo de la Issue
+#26. El MVP actual permanece sin cambios. La primera validación, cuando se
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)